bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/server-channel-groups.xml

```diff
@@ -52,6 +52,34 @@
 		</channels>
 	</channel-group-type>
 
+	<channel-group-type id="ftth">
+		<label>Connection FTTH Status</label>
+		<channels>
+			<channel id="sfp-present" typeId="sfp-present"/>
+			<channel id="sfp-alim-ok" typeId="sfp-alim-ok"/>
+			<channel id="sfp-has-power" typeId="sfp-has-power"/>
+			<channel id="sfp-has-signal" typeId="sfp-has-signal"/>
+			<channel id="link" typeId="link"/>
+			<channel id="sfp-pwr-tx" typeId="sfp-signal-level">
+				<label>TX Power</label>
+				<description>SFP Power in transmission</description>
+			</channel>
+			<channel id="sfp-pwr-rx" typeId="sfp-signal-level">
+				<label>RX Power</label>
+				<description>SFP Power in reception</description>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="xdsl">
+		<label>Connection xDSL status</label>
+		<channels>
+			<channel id="status" typeId="status"/>
```

Comment:
I will rather suggest a new channel type linked to a String and we will map `SynchroState`.
You can simply remove this channel and I will add it myself.

Reply:
Can you directly include your code modifications in your PR to this branch ? I will not be able to proceed by myself in the next coming days.

Reply:
You mean submit a PR to your branch ? Yes I can but in that case we will have to wait for another maintainer to merge "our" PR.
My initial idea was to merge your PR and then create another one to handle xDSL. Here is why I suggested to remove this channel.

Reply:
I'm going to proceed further with this PR and you'll add a second one.
The 'status' channel is temporary until you add yours. If there is not channel in the group it shows an error.

Reply:
Ok